deepface/DeepFace.py

```diff
@@ -298,7 +298,7 @@ def find(
             Set to False to avoid the exception for low-resolution images (default is True).
 
```

Comment:
you added some facial recognition models for yolo but they are missing here

Reply:
fixed

Reply:
there is no Yolo-Face in modelling